test/test_document_store.py

```diff
@@ -1633,6 +1636,151 @@ def test_DeepsetCloudDocumentStore_query(deepset_cloud_document_store):
     assert len(filtered_docs) < len(docs)
 
 
+@pytest.mark.parametrize(
+    "body, expected_count",
+    [
+        (
+            {
+                "data": [
+                    {
+                        "evaluation_set_id": uuid4(),
+                        "name": DC_TEST_INDEX,
```

Comment:
The variable `DC_TEST_INDEX` is a pipeline name, right? Why is the pipeline name used as the name for the evaluation set? Or am I missing something here?

Reply:
(its just a variable, but it might be confusing)

Reply:
This is an index name. As we discussed it with @tstadel the name of an eval set is the same as the index name.